libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -186,6 +186,8 @@
         SIGNED_INTO_JETPACK,
         PERFORMED_JETPACK_SIGN_IN_FROM_STATS_SCREEN,
         STATS_SELECTED_INSTALL_JETPACK,
+        STATS_CANCELED_INSTALL_JETPACK,
+        STATS_COMPLETED_INSTALL_JETPACK,
```

Comment:
It doesn't look like `STATS_COMPLETED_INSTALL_JETPACK` is ever used.

Reply:
Oh I used a wrong value for completed event, fixed now 👍